workflows/variantmtb.nf

```diff
@@ -227,17 +215,7 @@ workflow VARIANTMTB {
 }
 
 
-// Function that checks whether params.cgi_cancer_types contains the quotations ('') and isnt just a string.
-// If lonely string, adds the quotations, so that cancer types consisting of multiple words can be read by querynator
```

Comment:
Ah that explains my questions from above but then the 'Any cancer type' thing has to be handed differently.

Reply:
I refactored this, because I realized that putting single-quotes around the cancer variable `'$cancer'` does the same as this function and is a lot handier.
Is your "has to be handled differently" regarding functionality or style?
as for functionality, I can say it works and is covered by the base test ([test.config](https://github.com/qbic-pipelines/variantmtb/blob/f251b7da98ed76e4744cf9334aab343c7c7c42a9/conf/test.config#L23) uses [input.csv](https://github.com/qbic-pipelines/variantmtb/blob/dev/tests/csv/input.csv), which does not contain cancer information)

Specified cancer types are tested in the [full test](https://github.com/HomoPolyethylen/variantmtb/blob/dev/conf/test_full.config) using [input_cancer.csv](https://github.com/HomoPolyethylen/variantmtb/blob/dev/tests/csv/input_cancer.csv) and part of this PR.

Does this answer the question or did I misunderstand?